lib/src/list_mutations.dart

```diff
@@ -132,7 +132,8 @@ SourceEdit _appendToBlockList(
 }
 
 /// Formats [item] into a new node for block lists.
-String _formatNewBlock(YamlEditor yamlEdit, YamlList list, YamlNode item) {
+(String formatted, String indent) _formatNewBlock(
```

Comment:
Are you sure it wouldn't make more sense to return `listIndentation` ? Instead of returning the actual string.

Reply:
I'll investigate that. I did that for convenience. I didn't want to change so much in the code.